sympy/stats/stochastic_process_types.py

```diff
@@ -148,8 +153,8 @@ def joint_distribution(self, *args):
             return JointDistribution(*args)
         # TODO: Add tests for the below part of the method, when implementation of Bernoulli Process
```

Comment:
Have you completed this `TODO`?

Reply:
Yes I have added two types of tests for this

Reply:
Should I remove that?

Reply:
Please remove the `TODO` comment from here then.